media-sound/qpwgraph/qpwgraph-0.5.2.ebuild

```
@@ -0,0 +1,56 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="PipeWire Graph Qt GUI Interface"
+HOMEPAGE="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/rncbc/qpwgraph"
+else
+	SRC_URI="https://gitlab.freedesktop.org/rncbc/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2 -> ${P}.tar.bz2"
```

I don't see much point in renaming it:
```suggestion
	SRC_URI="https://gitlab.freedesktop.org/rncbc/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2"
```

It's easier to search for upstream archives etc without the rename.

---

done!